install/installer/pkg/components/ide-service/ide-configmap.json

```diff
@@ -19,6 +19,14 @@
           "{{.CodeHelperImage}}"
         ],
         "versions": [
+          {
+            "version": "1.89.0",
+            "image": "{{.Repository}}/ide/code:commit-8a412095311a2ee0460abca3a4461704c4533374",
+            "imageLayers": [
+              "{{.CodeWebExtensionImage}}",
+              "{{.CodeHelperImage}}"
```

Comment:
Unrelated to the PR but this images should be pinned too cc @filiptronicek

Reply:
https://linear.app/gitpod/issue/ENT-64